snapcraft/_store.py

```diff
@@ -64,6 +64,27 @@ def _get_data_from_snap_file(snap_path):
     return snap_yaml
 
 
+@contextmanager
+def _get_icon_from_snap_file(snap_path):
+    icon_file = None
+    with tempfile.TemporaryDirectory() as temp_dir:
+        output = subprocess.check_output(
+            ['unsquashfs', '-d',
+             os.path.join(temp_dir, 'squashfs-root'),
+             snap_path, '-e', 'meta/gui'])
+        logger.debug(output)
```

Comment:
I suspect this debug output will not be very useful on its own. Would it be worth adding some context to it?

Reply:
Yeah, makes sense, let me fix that.